api/src/main/migration.yml

```
@@ -1,9 +1,9 @@
-systemUnderAudit: nplinit
+$schema: https://documentation.noumenadigital.com/migration-schema-v2.yml
```

```suggestion
$schema: https://documentation.noumenadigital.com/schemas/migration-schema-v2.yml
```

Sorry, I thought it grabbed what was in my clip board but it seems it only copied the errored code as a starter (first time using suggestions)